Library/Formula/libvisio.rb

```diff
@@ -15,10 +14,11 @@ class Libvisio < Formula
   depends_on "pkg-config" => :build
   depends_on "boost" => :build
   depends_on "cppunit" => :build
-  depends_on "libwpd"
-  depends_on "libwpg"
-  depends_on "icu4c"
   depends_on "librevenge"
+  depends_on "icu4c"
+  depends_on "libxml2"
+  depends_on "homebrew/dupes/gperf"
+  depends_on "homebrew/dupes/zlib"
```

Comment:
It's fine for you to just commit them unsquashed but I can't add commits to this PR.